.circleci/template.yml

```
@@ -10,6 +10,17 @@ parameters:
     default: false
 
 commands:
+  halt_if_short_flow:
+    steps:
+      - run:
+          name: Check Commit Message
+          command: |
+            COMMIT_MESSAGE=$(git log -n 1 --pretty=format:%s ${CIRCLE_SHA1})
+            if [[ "$COMMIT_MESSAGE" == *"[short-flow]"* ]]; then
```

```suggestion
            COMMIT_MESSAGE=$(git log -n 1 --pretty=format:%s ${CIRCLE_SHA1})

            if [[ "$COMMIT_MESSAGE" == *"[short-flow]"* ]]; then
```

A blank line would help separate variables from instructions.